R/methods-epi_archive.R

```diff
@@ -171,8 +256,6 @@ epix_fill_through_version <- function(x, fill_versions_end,
 #'   as_epi_archive(compactify = TRUE)
 #' # merge results stored in a third object:
 #' xy <- epix_merge(x, y)
-#' # vs. mutating x to hold the merge result:
-#' x$merge(y)
 #'
 #' @importFrom data.table key set setkeyv
 #' @export
```

Comment:
question: would we want this as an S3 generic as well, or as an implementation of the `merge` generic?  Then people using `methods(, "epi_archive")` might get better results.

Reply:
Not sure. I've never used `methods(, "<class>")` to explore documentation. Let's just see how `epix_*` works out for now?

Reply:
Sure.  Side note: `methods(, "<class>")` or `sloop::{s3,s4}_methods_class()` (better than the former when s4 is involved in ways that confuse `methods()` or its users) is useful for exploring undocumented stuff --- often S3 implementations won't actually be exported, documented, etc.  Of course it's not comprehensive if there are regular functions that can also operate on the class.  I guess that was one benefit of R6, potentially having a sort of full list of functionality at your fingertips.